Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Operator/API/Driver.yaml

```diff
@@ -13,23 +13,9 @@ apis:
         - mbOffset: Int
         - mbDriverId: Text
         - mbMobileNumber: Text
+        - mbOperationHubId: Text
```

Comment:
@0utkarsh you've asked to add `mbDate` filter, but we already have `mbFrom` and `mbTo`, I think this is enough